### PR TITLE
Couple of issues fixed

### LINKED ISSUE #687: [feat] Update the Google logo on the homepage.
### Would you like to have a new feature? Please describe.

The Google logo on the homepage of the website is outdated. Update the google.svg file in the src/assets/home/company repository.

### How would this feature be beneficial for the project

Using the new logo will make the website look more updated.

### Additional context
Screenshot (554)
)

_No response_

### PR DESCRIPTION
**Pull request by Subgor8126**
## Fixes Issue #687 and #693

This PR fixes the following issues:

A typo was fixed on the Open Source courses section and the Google logo on the homepage was updated.

Another issue mentioned in the issue #693 was partially fixed in pull request #692

### CODE REVIEW

**Review by Chandraprakash-Darji — APPROVED** [review]

**Comment by Chandraprakash-Darji** [comment]
@AnuragThePathak one test failed check it once and also you added google logo but not not used it in work section

**Comment by Subgor8126** [comment]
Maybe there are no updates needed in the work section because the filename of the logo image is same as the previous one (google.svg).
I'm assuming that by work section you mean the Work.jsx file in src/components/PageComp/HomePage/

Please correct me if I'm wrong.

File: src/assets/home/company/google.svg
Comment (Chandraprakash-Darji) [inline src/assets/home/company/google.svg:1]: This should be svg, It is not rendering so Please re-check the file format

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by AnuragThePathak** [comment]
Screen Shot 2023-06-29 at 9 16 23 P
@Subgor8126 you need to fix it.

File: src/assets/home/company/google.svg
Comment (Chandraprakash-Darji) [reply src/assets/home/company/google.svg:1]: You have added the black logo on black background

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

File: src/assets/home/company/google.svg
Comment (Subgor8126) [reply src/assets/home/company/google.svg:1]: Done

**Comment by Subgor8126** [comment]
please check if the logo appears to be black now

**Comment by AnuragThePathak** [comment]
@Subgor8126 delete the latest commit, it has some problem so your workflow is failing (I assume it was just a simple merge, not resolving any conflicts).

**Comment by ItIsOHM** [comment]
Hey @Subgor8126 @AnuragThePathak @Chandraprakash-Darji, I'd like to work on this issue as a beginner contributor. May I start working on it?

**Comment by AnuragThePathak** [comment]
This is not an issue rather a PR. @ItIsOHM

**Comment by ItIsOHM** [comment]
> This is not an issue rather a PR. @ItIsOHM

Oh right I'm sorry about that, will ask in the issue chat.

**Comment by Subgor8126** [comment]
@ItIsOHM can we work together on this PR? I am a beginner as well.

**Comment by ItIsOHM** [comment]
> @ItIsOHM can we work together on this PR? I am a beginner as well.

Yea sure! Let me know here or anywhere else what's the status of this PR :D

**Comment by Subgor8126** [comment]
> > @ItIsOHM can we work together on this PR? I am a beginner as well.
> 
> Yea sure! Let me know here or anywhere else what's the status of this PR :D

It was probably a formatting issue. Can you please review the latest commit?

**Comment by Subgor8126** [comment]
@AnuragThePathak Please review the latest commit.

**Comment by AnuragThePathak** [comment]
@Subgor8126 it'll be reviewed by @Chandraprakash-Darji as of now. I do not have idea about svgs. I'll assign to someone else if he doesn't review in 2-3 days. Just remind me.

**Comment by Subgor8126** [comment]
> @Subgor8126 it'll be reviewed by @Chandraprakash-Darji as of now. I do not have idea about svgs. I'll assign to someone else if he doesn't review in 2-3 days. Just remind me.

OK, will do

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by AnuragThePathak** [comment]
@Subgor8126 your task is pretty simple now, Chandra has provided the svgs himself.

**Comment by Subgor8126** [comment]
@Chandraprakash-Darji please review the svg file now.

**Review by AnuragThePathak — APPROVED** [review]

**Pull request merged**